### PR TITLE
MINOR: update docs with regard to KIP-123

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

**Comment by mjsax** [comment]
Missed some changes in #3194 

Call for review @bbejeck @dguy @enothereska @miguno @guozhangwang

File: docs/streams.html [hunk 1]
```diff
@@ -825,15 +825,29 @@
...
         <ul>
-          <li> <code>timestamp.extractor</code> was deprecated and replaced by <code>default.timestamp.extractor</code>
-          <li> <code>key.serde</code> was deprecated and replaced by <code>default.key.serde</code>
-          <li> <code>value.serde</code> was deprecated and replaced by <code>default.value.serde</code>
+            <li> added overloads for <code>#stream()</code> that allow to define a <code>TimestampExtractor</code> per input stream </li>
+            <li> added overloads for <code>#table()</code> that allow to define a <code>TimestampExtractor</code> per input table </li>
+            <li> added overloads for <code>#globalKTable()</code> that allow to define a <code>TimestampExtractor</code> per global table </li>
         </ul>
 
         <h3><a id="streams_api_changes_01021" href="#streams_api_changes_01021">Notable changes in 0.10.2.1</a></h3>
```
Comment (guozhangwang) [inline docs/streams.html]: Should we change it to `Streams API changes in 0.10.2.1` also?

File: docs/streams.html [hunk 2]
```diff
@@ -825,15 +825,29 @@
...
+          <li> configuration parameter <code>timestamp.extractor</code> was deprecated and replaced by <code>default.timestamp.extractor</code>
+          <li> method <code>#keySerde()</code> was deprecated and replaced by <code>#defaultKeySerde()</code>
+          <li> method <code>#valueSerde()</code> was deprecated and replaced by <code>#defaultValueSerde()</code>
+          <li> new method <code>#defaultTimestampExtractor()</code> was added </code>
+        </ul>
+
+        <p> New methods in <code>TopologyBuilder</code>: </p>
+        <ul>
+            <li> added overloads for <code>#addSource()</code> that allow to define a <code>TimestampExtractor</code> per source node </li>
+            <li> added overloads for <code>#addGlobalStore()</code> that allow to define a <code>TimestampExtractor</code> per global store </li>
```
Comment (guozhangwang) [inline docs/streams.html]: nit: `per source node associated with the global store`?

File: docs/streams.html (hunk 1 above)
Comment (mjsax) [reply docs/streams.html]: I don't care. Not sure what other think. We only changes a default config -- "notable" sound resonable to me.

**Comment by mjsax** [comment]
Updated

File: docs/streams.html (hunk 1 above)
Comment (guozhangwang) [reply docs/streams.html]: I think either is fine, but better to be consistent: in 0.10.2.0 we say `Streams API` and in 0.10.2.1 we say `notable changes`. If you prefer `notable` changes let's make sure we use the same across all versions.

**Comment by mjsax** [comment]
Updated again.

**Comment by guozhangwang** [comment]
LGTM!

**Pull request closed without merging**